strain_gauge_calibration/CMakeLists.txt

```diff
@@ -0,0 +1,184 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(strain_gauge_calibration)
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED)
+
+## System dependencies are found with CMake's conventions
+find_package(catkin REQUIRED COMPONENTS
+  rospy
+  std_msgs
```

Comment:
std_msgs is not necessary

Catkin dependencies here should match with the catkin packages in packages.xml.

I this case:

```
rospy
diagnostic_msgs
```

as python-numpy is not a catkin package